src/generators/schema/images.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace Yoast\WP\SEO\Generators\Schema;
+
+use Yoast\WP\SEO\Values\Schema\Image;
+
+/**
+ * Returns ImageObject schema data for every image on the page.
+ */
+class Images extends Abstract_Schema_Piece {
+
+	/**
+	 * Generate image graph content.
+	 *
+	 * @return array The graph in array representation.
+	 */
+	public function generate() {
+		$graph = [];
+
+		$this->add_primary_image( $graph );
```

Comment:
I think we should not use a pass by ref here. I think it is more readable to just use $graph = $this